spec/helpers/testScheduler-ui.ts

```diff
@@ -0,0 +1,152 @@
+///<reference path='../../typings/main.d.ts'/>
+///<reference path='ambient.d.ts'/>
+
+import * as Rx from '../../dist/cjs/Rx.KitchenSink';
+import * as mocha from 'mocha';
+import * as Suite from 'mocha/lib/suite';
+import * as Test from 'mocha/lib/test';
+import * as commonInterface from 'mocha/lib/interfaces/common';
+import * as escapeRe from 'escape-string-regexp';
+import * as chai from 'chai';
+import * as sinonChai from 'sinon-chai';
+
+//setup sinon-chai
+chai.use(sinonChai);
+
+declare const module: any;
+declare const global: any;
+
+const diagramFunction = global.asDiagram;
+
+//mocha creates own global context per each test suite, simple patching to global won't deliver its context into test cases.
```

Comment:
If mocha is creating it's own global context, should we be adding `rxTestScheduler`, `hot`, `cold`, etc to this context? Or are you doing this already and I'm not seeing it?